### DISCUSSION TITLE
Save/load brightness and contrast settings

### DISCUSSION THREAD

**Discussion by Dennis182** [wkentaro/labelme#1419] [Q&A / Help]
Hi,

is there a possibility to save optimized brightness and contrast? In our current setting, we have to change the brightness and contrast for every image for proper segmentation. After we saved the annotation and reopen it we get the "bad" brightness and contrast. Is there a possibility to load/save optimized parameters like brightness and contrast?

Thank you in advance.

With kind regards
Dennis

**Comment by ryan-jeon** [comment]
Is there a solution to this yet? :D
